docs/install/install.md

```diff
@@ -138,37 +140,37 @@ Configuration can be done by both environment variable(conf/zeppelin-env.sh) and
     <td>ZEPPELIN_NOTEBOOK_HOMESCREEN</td>
     <td>zeppelin.notebook.homescreen</td>
     <td></td>
-    <td>Id of notebook to be displayed in homescreen ex) 2A94M5J1Z</td>
+    <td>A notebook id displayed in Zeppelin homescreen <br />i.e. 2A94M5J1Z</td>
   </tr>
   <tr>
     <td>ZEPPELIN_NOTEBOOK_HOMESCREEN_HIDE</td>
     <td>zeppelin.notebook.homescreen.hide</td>
     <td>false</td>
-    <td>hide homescreen notebook from list when this value set to "true"</td>
+    <td>This value can be "true" when you want to hide notebooks from Zeppelin homescreen.</td>
```

Comment:
actually, I think this is for hiding only the notebook with the id set with ZEPPELIN_NOTEBOOK_HOMESCREEN above.

So perhaps "when to hide the notebook id set by ZEPPELIN_NOTEBOOK_HOMESCREEN on the Zeppelin homescreen"